tests/pyfunc/test_scoring_server.py

```diff
@@ -346,6 +422,70 @@ def test_parse_with_schema(pandas_df_with_all_types):
     assert all(df["bad_boolean"] == [True, False, True])
 
 
+def test_parse_tf_serving_input():
+    # instances are correctly aggregated to dict of input name -> tensor
+    tfserving_input = {
+        "instances": [
+            {"a": "s1", "b": 1, "c": [1, 2, 3]},
+            {"a": "s2", "b": 2, "c": [4, 5, 6]},
+            {"a": "s3", "b": 3, "c": [7, 8, 9]},
+        ]
+    }
+    result = pyfunc_scoring_server.parse_tf_serving_input(tfserving_input)
+    assert (result["a"] == np.array(["s1", "s2", "s3"])).all()
+    assert (result["b"] == np.array([1, 2, 3])).all()
+    assert (result["c"] == np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])).all()
+
+    # input is bad if a column value is missing for a row/instance
+    tfserving_input = {
+        "instances": [
+            {"a": "s1", "b": 1},
+            {"a": "s2", "b": 2, "c": [4, 5, 6]},
+            {"a": "s3", "b": 3, "c": [7, 8, 9]},
+        ]
+    }
+    with pytest.raises(MlflowException) as ex:
+        pyfunc_scoring_server.parse_tf_serving_input(tfserving_input)
+    assert "The length of values for each input/column name are not the same" in str(ex)
```

Comment:
```suggestion
    with pytest.raises(MlflowException, match="The length of values for each input/column name are not the same"):
        pyfunc_scoring_server.parse_tf_serving_input(tfserving_input)
```

Can we use a `match` argument ([ref](https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions)) here? I know we're using a bare pytest-raises (= `pytest.raises` without `match`) in many tests. I'm planning to fix this.